circuits/integration/zk_circuits/valid_match_mpc.rs

```diff
@@ -336,356 +336,8 @@ async fn test_valid_match__non_integral_price(test_args: IntegrationTestArgs) ->
-    let party_id = fabric.party_id();
-    let price = *DUMMY_PRICE;
-    let my_order = create_test_order(party_id);
-    let mut my_balance = create_test_balance(party_id);
-    let amount = compute_max_amount(price, &my_order, &my_balance);
-
-    // Reduce the balance to be less than the amount
-    if my_balance.mint == my_order.base_mint {
-        my_balance.amount = 1;
-    }
-
-    // Validate that the constraints are not satisfied
-    let witness = setup_witness(
-        price,
-        amount,
-        my_order,
-        my_balance,
-        test_args.mpc_fabric.clone(),
-    )
-    .await;
-    if constraints_satisfied(witness, test_args.mpc_fabric.clone()).await? {
-        return Err(eyre!("Constraints satisfied on invalid witness"));
-    }
-
-    Ok(())
-}
-
-/// Test the case in which the matched amount exceeds the order size for a party
-async fn test_valid_match__amount_exceeds_order(test_args: IntegrationTestArgs) -> Result<()> {
-    let fabric = &test_args.mpc_fabric;
-    let party_id = fabric.party_id();
-    let price = *DUMMY_PRICE;
-    let my_order = create_test_order(party_id);
-    let my_balance = create_test_balance(party_id);
-
-    // Both parties give an excessive amount so that the minimum amount certainly
-    // exceeds the order size
-    let amount = my_order.amount + 1;
-
-    // Validate that the constraints are not satisfied
-    let witness = setup_witness(
-        price,
-        amount,
-        my_order,
-        my_balance,
-        test_args.mpc_fabric.clone(),
-    )
-    .await;
-    if constraints_satisfied(witness, test_args.mpc_fabric.clone()).await? {
-        return Err(eyre!("Constraints satisfied on invalid witness"));
-    }
-
-    Ok(())
-}
-
-/// Test the case in which the `max_minus_min` field is incorrectly computed
-async fn test_valid_match__incorrect_max_minus_min(test_args: IntegrationTestArgs) -> Result<()> {
-    let fabric = &test_args.mpc_fabric;
-    let party_id = fabric.party_id();
-    let price = *DUMMY_PRICE;
-    let my_order = create_test_order(party_id);
-    let my_balance = create_test_balance(party_id);
-    let amount = compute_max_amount(price, &my_order, &my_balance);
-
-    // Validate that the constraints are not satisfied
-    let mut witness = setup_witness(
-        price,
-        amount,
-        my_order,
-        my_balance,
-        test_args.mpc_fabric.clone(),
-    )
-    .await;
-    let mut rng = thread_rng();
-    witness.match_res.max_minus_min_amount = Scalar::random(&mut rng)
-        .to_linkable()
-        .allocate(PARTY0, fabric);
-
-    if constraints_satisfied(witness, test_args.mpc_fabric.clone()).await? {
-        return Err(eyre!("Constraints satisfied on invalid witness"));
-    }
-
-    Ok(())
-}
-
-/// Test the case in which the `max_minus_min` field is switched to be
-/// `min - max` (i.e. negative)
-async fn test_valid_match__max_minus_min_negative(test_args: IntegrationTestArgs) -> Result<()> {
```

Comment:
Seems we didn't port over this test, do we not need it?

Reply:
correct